packages/react/src/stateProxy.ts

```
@@ -0,0 +1,113 @@
+import type { Errors, State } from '@clerk/types';
+
+import type { IsomorphicClerk } from './isomorphicClerk';
+import { inBrowser } from '@clerk/shared';
```

ESLint

[error] '@clerk/shared' import is restricted from being used. Please always import from '@clerk/shared/<module>' instead of '@clerk/shared'.

(no-restricted-imports)